packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -956,4 +976,32 @@ export class VerticalStackedBarChartBase extends React.Component<
     const yValue = point.yAxisCalloutData || point.data;
     return point.callOutAccessibilityData?.ariaLabel || `${xValue}. ${legend}, ${yValue}.`;
   };
```

Comment:
This function is same for VerticalBarChart and VerticalStackedBarChart but a little different for GroupedVerticalBarChart. Extracting it as a utility function may become complex because it contains many references to the class members.